src/asmcnc/apps/drywall_cutter_app/config/config_loader.py

```diff
@@ -1,17 +1,20 @@
 import json
 import os
+import inspect
 
+import config_classes
 from asmcnc.comms.logging_system.logging_system import Logger
 
-import config_classes
-import inspect
 
-current_dir = os.path.dirname(os.path.realpath(__file__))
-configurations_dir = os.path.join(current_dir, 'configurations')
-cutters_dir = os.path.join(current_dir, 'cutters')
-temp_dir = os.path.join(current_dir, 'temp')
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Why not use path_utils here?

Reply:
Some of these paths won't exist, I'd prefer to build them like this. Also it was like this before path_utils existed I just renamed them.